docs/src/cli/commands.md

```diff
@@ -93,7 +93,11 @@ of all workspace programs before running them.
 If the configured network is a localnet, then automatically starts the localnetwork and runs
 the test.
 
-When running tests we stream program logs to .anchor/program-logs/<address>.<program-name>.log
+::: tip Note
+Be sure to shutdown any other local validators, otherwise `anchor test` will fail to run.
```

Comment:
```suggestion
Be sure to shutdown any other local validators, otherwise `anchor test` will fail to run.
If you'd like to run tests against a local validator started manually, run `anchor test --skip-local-validator`.
```